react-accordion/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import { TopicCard } from './TopicCard';
+
+type Topic = {
+  id: number;
+  title: string;
+  content: string;
+};
+type Props = {
+  topics: Topic[];
+};
+export function Accordion({ topics }: Props) {
+  const [activeId, setActiveId] = useState(0);
+  return topics.map((topic) => (
```

Comment:
It doesn't look to me like this will close the activeId if you click it's title while it is open. Try it and see.

Click a title to open it, then click the title again. It should close.